oidc-controller/api/core/config.py

```diff
@@ -39,6 +39,8 @@ class GlobalConfig(BaseSettings):
     CONTROLLER_URL: str = os.environ.get("CONTROLLER_URL")
     # if ngrok is blocked by your local network, set this to your localhost for testing.
     CONTROLLER_URL_LOCAL: str = os.environ.get("CONTROLLER_URL_LOCAL", CONTROLLER_URL)
+    # Where to send users when trying to scan with their mobile camera (not a wallet)
+    CONTROLLER_CAMERA_REDIRECT_URL: str = os.environ.get("CONTROLLER_CAMERA_REDIRECT_URL")
```

Comment:
Could we provide a default, built-in page for users who don't have a separately-hosted page?

I like the fact any web-hosted page could be used, but I think it would be nice if we did not have to necessarily rely on that for the functionality.

Reply:
I like this idea.
Are you thinking like a bare bones description page? Or should we do a little UI/UX exercise?

Reply:
I think, for now, we can have a pretty lightweight "it looks like you didn't use a wallet, try again" type of page. Styled so it is not terrible, but basic. We can do UI/UX as a second, separate step.